README.md

```diff
@@ -49,6 +49,10 @@ Short description.
 
   Mobile app for food-sharing, aiming to reduce food waste. It does this by connecting those with surplus food to those who need or wish to consume such food. The food must be edible; it can be raw or cooked, sealed or open.
 
+- [Peckas Tomater](https://peckas.com) | Sweden | 2015
```

Comment:
Hey @JohanVikman, I tried to check the URL but seems like it's not working. I tried to check the Swedish version (https://peckas.se) and switch to English, and seems like the English version of the site is at the following URL:

```suggestion
- [Peckas Tomater](http://www.peckas.com) | Sweden | 2015
```